resources/views/buyers/checkout_complete.blade.php

```diff
@@ -0,0 +1,38 @@
+@extends('layouts.app')
+
+@section('title', 'Buyer: Checkout Complete Page')
+
+@section('content')
+    <style>
+        ::marker {
+            color: #071657;
+            font-size: 300%;
+        }
+
+        ul {
+            font-size: 16px;
+            line-height: 30px;
+            list-style: square;
+        }
+
+        ul li span {
+            vertical-align: 10px;
+        }
+    </style>
```

Comment:
This one as well

Reply:
I put this in checkout.css with commit [72317eb](https://github.com/kredo-school/16th_rankshty/pull/45/commits/72317ebe156c5527809ac9d84121c000bbc9037d).